.homeycompose/capabilities/operation_mode_zone.json

```diff
@@ -13,22 +16,31 @@
     {
       "id": "0",
       "title": {
-        "en": "Heat room",
-        "fr": "Chauffer la pièce"
+        "en": "Heat the room",
+        "fr": "Chauffer la pièce",
+        "nl": "Verwarm de ruimte",
+        "sv": "Värma upp rummet",
+        "es": "Calentar la habitación"
       }
     },
     {
       "id": "1",
       "title": {
-        "en": "Heat flow",
-        "fr": "Flux de chauffage"
+        "en": "Heat the flow",
+        "fr": "Chauffer l'écoulement",
+        "nl": "Verwarm de doorstroming",
+        "sv": "Värma upp flödet",
```

Comment:
"Framledningstemperatur" is the correct word to use.
Could even be "Fast framledningstemperatur" as the set point of the heat flow is a fixed temperatur, fixed as in "fast".
Maybe "fast" would make people understand that the chosen temperature is a fixed chosen temperature. Then again, only choosing "Framledningstemperatur" is more clean.